packages/terra-clinical-result/src/flowsheet-result-cell/FlowsheetResultCell.jsx

```diff
@@ -1,4 +1,4 @@
-import React, { useRef, useState, useEffect } from 'react';
+import React, { useRef, useState, useLayoutEffect } from 'react';
```

Comment:
Fixes in this file are related to the flicker defect being dealt with in this PR: https://github.com/cerner/terra-clinical/pull/881

Adding them to this PR because otherwise we'll have the same issue.